encoder/vaapiencpicture.h

```diff
@@ -124,6 +124,8 @@ createMiscObject(VAEncMiscParameterType miscType, T * &bufPtr)
     if (obj) {
         misc->type = miscType;
         bufPtr = (T *) (misc->data);
+        if (bufPtr)
```

Comment:
@xuguangxin misc is an input of createBufferObject,  and drive will init misc->data, may change it to NULL.  So we should check bufPtr